qtop.py

```diff
@@ -908,17 +935,9 @@ def load_yaml_config():
     $HOME/.local/qtop/
     in that order.
     """
-    config = read_yaml_natively(os.path.join(QTOPPATH, QTOPCONF_YAML))
+    config = read_yaml_natively(os.path.join(os.path.realpath(QTOPPATH), QTOPCONF_YAML))
```

Comment:
imported 2-3 really uniquely-named functions from os.path (realpath, expandvars, getmtime), otherwise it will be chaos!!